static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -637,23 +638,28 @@ const buildEpicCopy = (
         ? getCountryName(geolocation)
         : undefined;
 
-    const replaceCountryNameAndArticlesViewed = (s: string): string =>
-        replaceArticlesViewed(
-            replaceCountryName(s, countryName),
-            articlesViewedCount
+    const localCurrencySymbol = getLocalCurrencySymbol(geolocation);
+    const replaceCurrencySymbol = (s: string): string => s.replace(
+        /%%CURRENCY_SYMBOL%%/g,
+        localCurrencySymbol,
+    );
+
+    const replaceTemplates = (s: string): string =>
```

Comment:
I realised we're not currently replacing the currency template in the body copy. This is necessary now that we're moving away from using the `highlightedText` field for liveblog epics.